appium/options/base_options.py

```diff
@@ -0,0 +1,34 @@
+from typing import Any, TypeVar, Generic, Type, Callable, Optional
```

Comment:
not sure about the name of this module. it contains the descriptor class, but is called base_options

Reply:
Changed the name of the module to `base_options_descriptor`